content/articles/faq-ssl-certificates.markdown

```diff
@@ -42,6 +42,13 @@ categories:
     An SSL certificate is not a single-time purchase. You need to renew the certificate periodically, sometimes you need to rekey it or you find yourself having issues installing it, and you want to ask for some help. All these tasks are covered by our subscription service.
 
     Additionally, if you are requesting a [Let's Encrypt certificate](/articles/letsencrypt/), we will use our DNS hosting service to automatically provision the DNS records required to validate your certificate, and we will [automatically renew](/articles/letsencrypt/#auto-renewal) the certificate for you before it expires. All these services require a subscription.
+
+1.  #### I'm unsubscribed/ing my account, will my current certificate keep valid?
```

Comment:
The _"unsubscribed/ing"_ is something that I don't think we have done before. Maybe you could better phrase this as: 
> Will my certificate remain valid after unsubscribing?

Reply:
Sounds better, thanks. 797dee9a259541d62773e0393843d4c6ba3f0087